.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,59 @@
+name: Node.js CI
+
+on: [push, pull_request]
```

Comment:
Could this simply be `on: pull_request`? (See my comment above)

Reply:
We definitely need this to run on some pushes so that semantic-release will run. This is primarily for when it runs on the master branch, but with semantic-release v16 it could be on branches with other names so long as those were configured such that semantic-release would perform releases on those branches.

Reply:
Then how about
```
on:
  push:
    branches:
      - master
  pull_request:
```

(see [reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows))